components/button/Copy.js

```diff
@@ -20,7 +21,7 @@ const Copy = ({ value, className = '' }) => {
         <Button
             onClick={handleClick}
             icon="clipboard"
-            className={`${className} ${copied ? 'copied' : ''}`}
+            className={classnames([className, copied ? 'copied' : ''])}
```

Comment:
what you wrote certainly works. But just to have a more homogeneous code, could it be:
```
className={classnames([className, copied && 'copied'])}
```
?
We're using that syntax in other parts of the code. Sorry for the annoying request ;)